### DISCUSSION TITLE
Lava Optimization 0.2.3

### DISCUSSION THREAD

**Discussion by PhilippPlank** [lava-nc/lava-optimization#152] [Announcements]
# [Lava Optimization v0.2.3](https://github.com/lava-nc/lava-optimization) Release Notes
December 14, 2022

## New Features and Improvements

- New neuron model for OptimizationSolver provides substantial performance improvements on QUBO problems. ([PR #138](https://github.com/lava-nc/lava-optimization/pull/138) & [PR #141](https://github.com/lava-nc/lava-optimization/pull/141) & [PR #145](https://github.com/lava-nc/lava-optimization/pull/145) & [PR #146](https://github.com/lava-nc/lava-optimization/pull/146)).
- Improved SolverTuner interface and functionality, allowing for arbitrary hyper-parameters search spaces and stopping conditions. ([PR #123](https://github.com/lava-nc/lava-optimization/pull/123))
- Improved profiling utils can now measure energy consumption of OptimizationSolver running on Loihi 2. ([PR #131 in lava](https://github.com/lava-nc/lava/pull/131) & [PR #137 in lava](https://github.com/lava-nc/lava/pull/137))

## Bug Fixes and Other Changes

- Updated docstrings for Bayesian solver. ([PR #130 in lava](https://github.com/lava-nc/lava/pull/130))
- Fixed typos in documentation. ([PR #134 in lava](https://github.com/lava-nc/lava/pull/134))
- Automatically cancel older CI runs of a PR if a newer one was started due to a push. ([PR #135 in lava](https://github.com/lava-nc/lava/pull/135))
- Removed unused pre- and post- hook calls in the OptimizationSolver. ([PR #144 in lava](https://github.com/lava-nc/lava/pull/144))

## Breaking Changes

- The SolverTuner API has been updated.
- The QUBO solver now requires a different set of hyperparameters.

## Known Issues

- No known issues in this release.

## Thanks to our Contributors

- Intel Labs Lava Developers
- Georg Reich @1b15 made their first contribution in https://github.com/lava-nc/lava-optimization/pull/134

**Full Changelog**: https://github.com/lava-nc/lava-optimization/compare/v0.2.2...v0.2.3

<hr /><em>This discussion was created from the release <a href='https://github.com/lava-nc/lava-optimization/releases/tag/v0.2.3'>Lava Optimization 0.2.3</a>.</em>

**Comment by hyydw** [comment]
已收到！
